app/views/shared/_next_event.html.erb

```diff
@@ -4,7 +4,11 @@
     <h2 class="font-bold tracking-tight text-gray-900 text-2xl text-justify">
       <%= link_to @next_event.title, @next_event %>
     </h2>
-    <h3 class="text-gray-600">Over <%= time_ago_in_words(@next_event.date) %></h3>
+    <% if @next_event.past? %>
+      <h3 class="text-gray-600">Activiteit is bezig</h3>
+    <% else %>
+      <h3 class="text-gray-600">Over <%= time_ago_in_words(@next_event.date) %></h3>
+    <% end %>
```

Comment:
Not sure if this works, but it is probably neater to do it inline and not redefine the styling:
```suggestion
    <h3 class="text-gray-600">
      <%= @next_event.past ? "Activiteit is bezig" : "Over #{time_ago_in_words(@next_event.date)}" %>
    </h3>
```